sbncode/CAFMaker/FillReco.cxx

```diff
@@ -460,20 +497,34 @@ namespace caf
     srtrack.end.y = track.End().Y();
     srtrack.end.z = track.End().Z();
 
-    srtrack.ID = particle.Self();
-    srtrack.slcID = (primary) ? primary->Self() : -1;
+  }
+
+  void FillPFPVars(const recob::PFParticle &particle,
```

Comment:
PFP is "particle flow particle"? I'm not sure we need to let that jargon leak into the CAFs. Something like `SRLineage`, but less lame? If it's just the name of the type maybe it doesn't matter so much...

Reply:
Yeah, this is a bunch of information from Pandora. Not all of it is hierarchy (e.g. there is a score of how track-like a given particle is) so I thought this was the best solution, but I do not feel particularly strongly